djautotask/tests/test_commands.py

```diff
@@ -85,32 +85,59 @@ def test_full_sync(self):
         self.assertEqual(msg, out.getvalue().strip())
 
 
+class AbstractPicklistSyncCommandTest(AbstractBaseSyncTest):
+
+    def get_return_value(self, at_object, fixture_list):
+        field_info = fixture_utils.generate_picklist_objects(
+            self.field_name, fixture_list)
+
+        return field_info
+
+
 class TestSyncTicketCommand(AbstractBaseSyncTest, TestCase):
     command_name = 'ticket'
     args = (
-        mocks.service_ticket_api_call,
+        mocks.ticket_api_call,
         fixtures.API_SERVICE_TICKET_LIST,
-        'ticket',
+        command_name,
     )
 
     def setUp(self):
         super().setUp()
         fixture_utils.init_ticket_statuses()
 
 
-class TestSyncTicketStatusCommand(AbstractBaseSyncTest, TestCase):
+class TestSyncTicketStatusCommand(AbstractPicklistSyncCommandTest, TestCase):
     command_name = 'ticket_status'
+    field_name = 'Status'
 
-    def get_return_value(self, at_object, fixture_list):
-        field_info = fixture_utils.generate_picklist_objects(
-            'Status', fixture_list)
+    args = (
+        mocks.ticket_status_api_call,
+        fixtures.API_TICKET_STATUS_LIST,
+        command_name,
 
-        return field_info
+    )
+
+
+class TestSyncTicketPriorityCommand(AbstractPicklistSyncCommandTest, TestCase):
+    command_name = 'ticket_priority'
+    field_name = 'Priority'
 
     args = (
-        mocks.service_ticket_status_api_call,
-        fixtures.API_TICKET_STATUS_LIST,
-        'ticket_status',
+        mocks.ticket_priority_api_call,
+        fixtures.API_TICKET_PRIORITY_LIST,
+        command_name,
+    )
+
+
+class TestSyncQueueCommand(AbstractPicklistSyncCommandTest, TestCase):
+    command_name = 'queue'
```

Comment:
There doesn't seem to be a good reason to assign the command name here. I would expect this to be called somewhere like `cls` or `self`, but I can't find any usages anywhere besides in the `args` assignment. 

I also don't see this pattern in django connectwise. So take this out from all the models, unless you do have a good reason for this and I am just not seeing it.

Reply:
Hmm, I think that was a leftover that should've been removed. I can't recall the reason for it but I resolved another way and never removed the `command_name` field. Thanks for spotting it. Fixed now